serial_server.py

```diff
@@ -232,14 +233,14 @@ def bytesize(self, c, data=None):
     def parity(self, c, data=None):
         """Sets the parity."""
         ser = self.getPort(c)
-        bsizes = [x[1] for x in ser.getSupportedParities()]
+        bsizes = [x[1] for x in ser.PARITIES]
         if data is None:
```

Comment:
Again, no need for the list comprehension, ser.PARITIES is just a sequence of strings.